src/components/atoms/button.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function Button(props) {
+  return (
+    <button
+      className="btn btn-primary"
+      onClick={() => props.handleClick()}
```

Comment:
Could probably be simplified to
```suggestion
      onClick={props.handleClick}
```

Reply:
fixed